evap/staff/views.py

```diff
@@ -651,16 +653,21 @@ def notify_reward_points(grantings, **_kwargs):
 
         if course.state in ['evaluated', 'reviewed'] and course.is_in_evaluation_period:
             course.reopen_evaluation()
-        if form.has_changed():
-            form.save(user=request.user)
-        if formset.has_changed():
-            formset.save()
+
+        form_has_changed = form.has_changed() or formset.has_changed()
+
+        if form_has_changed:
```

Comment:
i would prefer this variable to be inlined, because it doesn't add much value and is more text to parse, but you don't need to change it.

Reply:
`form_has_changed` is reused later (l.667), that's why I did not inline it

Reply:
oh, right.